target/arm/ptw.c

```diff
@@ -438,6 +438,7 @@ static inline int
 pte_to_sprr_prot_is_guarded(CPUARMState *env, int ap, int xn, int pxn, bool guarded)
 {
     int sprr_idx = ((ap << 2) | (xn << 1) | pxn) & 0xf;
+    assert(arm_current_el(env) < 2);
```

Comment:
Why?

Reply:
That's the reason:
uint64_t sprr_el_br_el1[4][2];

A security measure to prevent it overflowing.